renku/cli/init.py

```diff
@@ -236,31 +256,39 @@ def check_git_user_config():
     callback=validate_name,
     help='Provide a custom project name.',
 )
-@click.option('--template-id', help='Provide the id of the template to use.')
 @click.option(
+    '-i', '--template-id', help='Provide the id of the template to use.'
+)
```

Comment:
These short options are good but a bit inconsistent which makes them hard to remember and kinda defeat the purpose. May I suggest that we use `-t` for `--template-id` since it should be used most, and then use `-i` for `--template-index` (which will be consistent with the reset)?

Reply:
Good point! Thanks for considering the consistency with other commands here. I changed it as suggested